category_filter.py

```diff
@@ -127,9 +128,10 @@ def get_compression_metadata(self, property_output_name: str) -> Dict:
             if prop[0] == base_output_name:
                 compression = prop[6]
         if compression is None:
-            return {"Lossy Compression Algorithm": "None", "Is Compressed": False}
+
+            return {"Lossy compression filter": np.bytes_("None".encode('utf-8')), "Is Compressed": False}
         else:
-            return {"Lossy Compression Algorithm": compression, "Is Compressed": False}
+            return {"Lossy compression filter": np.bytes_(compression.encode('utf-8')), "Is Compressed": False}
```

Comment:
Why do this..?

Reply:
I have to force SOAP to write the string as bytes or else https://github.com/SWIFTSIM/swiftsimio/blob/fda13e2a7a704a37c6a9e26bfb7fee23ffc4b33f/swiftsimio/reader.py#L959 complains

Reply:
Cool, I'm happy to do it that way